src/DependencyInjection/Configuration.php

```diff
@@ -37,8 +39,15 @@ private function addServersSection($rootNode)
                     ->useAttributeAsKey('alias', false)
                     ->prototype('array')
                         ->children()
-                            ->scalarNode('ip')->defaultValue('localhost')->end()
+                            ->scalarNode('host')->defaultValue('localhost')->end()
                             ->scalarNode('port')->defaultValue(6379)->end()
+                            ->integerNode('database')->defaultValue(0)->end()
+                            ->scalarNode('scheme')->defaultValue('tcp')->end()
+                            ->booleanNode('async_connect')->defaultValue('false')->end()
+                            ->booleanNode('persistent')->defaultValue('false')->end()
+                            ->floatNode('timeout')->defaultValue(10)->min(0)->end()
+                            ->floatNode('read_write_timeout')->min(0)->end()
```

Comment:
:warning: The documentation mention `readwritetimeout` in the client section not `read_write_timeout` in the server section, which and where is it?
Same goes for the other options, some of them are not even mentioned in the doc,
 maybe we should?

Reply:
i updated from readwritetimeout to read_write_timeout. i will update the doc too